test/e2e/servers/graphql-main-server/no-auth-requests.spec.ts

```diff
@@ -213,10 +206,10 @@ describe("graphql", () => {
       await testRateLimitLoginByIp(args)
     })
 
-    it("rate limits too many invalid login requests by IP, wrong phone", async () => {
+    it.skip("rate limits too many invalid login requests by IP, wrong phone", async () => {
```

Comment:
if we will skip it then there is no need to change expectedMessage, also if this will be handled by twilio then we can remove it

Reply:
the current issue on this one is this can't be tested. 

we can't have an invalid phone without plugging twilio. 

otherwise the only valid phone in the test will be in the galoy.yaml. they are by definition valid. they go through this branch https://github.com/GaloyMoney/galoy/pull/2029/files#diff-853c250e67d6440cae6da35382ad50102137342a2a3aed619ba4e7687eff8bc9R184